src/EnergyPlus/SizingManager.cc

```diff
@@ -770,6 +782,778 @@ namespace SizingManager {
+						DataSizing::VdzHtgByZone( SingleDuct::SysATMixer( aTMixerNum ).ZoneNum ) = SingleDuct::SysATMixer( aTMixerNum ).DesignPrimaryAirVolRate;
+						DataSizing::VdzMinHtgByZone( SingleDuct::SysATMixer( aTMixerNum ).ZoneNum ) = SingleDuct::SysATMixer( aTMixerNum ).DesignPrimaryAirVolRate;
+					}
+				}
+			}
+
+			std::string curName = FinalSysSizing( AirLoopNum ).AirPriLoopName;
+			ReportSizingManager::ReportSizingOutput( "AirLoopHVAC", curName, "Sum of Air Terminal Maximum Heating Flow Rates [m3/s]", airLoopHeatingMaximumFlowRateSum );
+			ReportSizingManager::ReportSizingOutput( "AirLoopHVAC", curName, "Sum of Air Terminal Minimum Heating Flow Rates [m3/s]", airLoopHeatingMinimumFlowRateSum );
+			ReportSizingManager::ReportSizingOutput( "AirLoopHVAC", curName, "Sum of Air Terminal Maximum Flow Rates [m3/s]", airLoopMaxFlowRateSum );
+
+			//Adjust system sizing info
+			if ( allocated( FinalSysSizing ) ) {
+				// correct sizing design heating volume flow rate based on finalized air terminal unit operation
+
+				if ( FinalSysSizing( AirLoopNum ).SizingOption == NonCoincident ) { // If non-coincident sizing method for this air loop, the we can use these sum's from air terminals directly
+					FinalSysSizing( AirLoopNum ).DesHeatVolFlow = max( airLoopHeatingMaximumFlowRateSum, FinalSysSizing( AirLoopNum ).DesHeatVolFlow );
+					FinalSysSizing( AirLoopNum ).DesMainVolFlow = max( airLoopMaxFlowRateSum, FinalSysSizing( AirLoopNum ).DesMainVolFlow );
+					if ( FinalSysSizing( AirLoopNum ).sysSizeCoolingDominant ) {
+						FinalSysSizing( AirLoopNum ).DesCoolVolFlow = FinalSysSizing( AirLoopNum ).DesMainVolFlow;
+						FinalSysSizing( AirLoopNum ).MassFlowAtCoolPeak = FinalSysSizing( AirLoopNum ).DesCoolVolFlow * DataEnvironment::StdRhoAir ;
+					} else if ( FinalSysSizing( AirLoopNum ).sysSizeHeatingDominant ) { //make sure cooling is at least at minimum.
+						FinalSysSizing( AirLoopNum ).DesCoolVolFlow =  max( airLoopHeatingMinimumFlowRateSum, FinalSysSizing( AirLoopNum ).DesCoolVolFlow );
+						FinalSysSizing( AirLoopNum ).MassFlowAtCoolPeak = FinalSysSizing( AirLoopNum ).DesCoolVolFlow * DataEnvironment::StdRhoAir ;
+					}
+				} else if ( FinalSysSizing( AirLoopNum ).SizingOption == Coincident ) {
+
+					if ( FinalSysSizing( AirLoopNum ).sysSizeCoolingDominant ) { // use minimum heating flow sum from air terminals 
+						// know that minimum heating flow is a hard minimum regardless of concurrence situation, so make sure that design is at least that high.
+						FinalSysSizing( AirLoopNum ).DesHeatVolFlow = max( airLoopHeatingMinimumFlowRateSum, FinalSysSizing( AirLoopNum ).DesHeatVolFlow );
+						FinalSysSizing( AirLoopNum ).DesMainVolFlow = max( airLoopHeatingMinimumFlowRateSum, FinalSysSizing( AirLoopNum ).DesMainVolFlow );
+						FinalSysSizing( AirLoopNum ).DesCoolVolFlow = FinalSysSizing( AirLoopNum ).DesMainVolFlow;
```

Comment:
Cooling is different here than heating and main. Can't recall right now what Cooling and Heating is used for in the air loop simulation. Thought it was always main.

Reply:
That is, different compared to calculation in heating dominant.